frontend/src/ui/OnetLink.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export interface OnetLinkProps {
+  socCode: string;
+}
+
+const OnetLink = (props: OnetLinkProps) => {
+  let newcode = props.socCode;
+  if (!newcode.endsWith('.00')) {
+    newcode = `${newcode}.00`;
+  }
+  return (
+    <React.Fragment>
```

Comment:
nit: you can remove the `<React.Fragment>` (also written as `<>`). Fragment is used when you want to return sibling components at the top level of a component, e.g.

```
<>
  <div>section 1</div>
  <div>section 2</div>
</>
```

Reply:
fixing. ty